### PR TITLE
Update gulpfile.mjs

### PR DESCRIPTION
**Pull request by dsarkar**
update current app versions

### CODE REVIEW

File: gulpfile.mjs [hunk 1]
```diff
@@ -525,11 +525,11 @@ function replaceVersionNumbers() {
     .src([PATHS.dist + '/**/*.html', PATHS.dist + '/**/*.json'])
     .pipe(replace('[ios.minimum-required-os-version]', '12.5'))
     .pipe(replace('[ios.minimum-app-version]', '1.5.3'))
-    .pipe(replace('[ios.current-app-version]', '3.0.0'))
+    .pipe(replace('[ios.current-app-version]', '3.0.1'))
     .pipe(replace('[android.latest-os-version]', '13'))
     .pipe(replace('[android.minimum-required-os-version]', '6'))
     .pipe(replace('[android.minimum-app-version]', '1.0.4'))
-    .pipe(replace('[android.current-app-version]', '3.0.1'))
+    .pipe(replace('[android.current-app-version]', '3.0.2'))
```
Comment (Ein-Tim) [inline gulpfile.mjs:532]: There was no release information / GitHub release for that version yet. Is this planned for later today?

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I am already seeing CWA Android updated to 3.0.2 on my phone.

**Comment by Ein-Tim** [comment]
@MikeMcC399 Yes, same here, but I'm missing the GitHub release. 🥲

But it's probably better to open an issue in the cwa-android repo reg. this.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> But it's probably better to open an issue in the cwa-android repo reg. this.

There is no action for the website. If you have concerns, then you should bring them up over there. You might however just wait until Monday for that.

**Pull request merged**